services/web/server/src/simcore_service_webserver/products_db.py

```diff
@@ -68,3 +68,13 @@ async def get_product_template_content(
                 .where(products.c.name == product_name)
             )
             return f"{content}" if content else None
+
+    async def get_product_max_number_studies_open_per_user(
```

Comment:
Would it make sense to have a a TTL cache here? I'd expect this value to rarely change

Reply:
I do not think that is a capital issue at the moment since this is only called in project opening call. until we profile I would not do optimizations.

Reply:
I see both sides, I am fine with not doing this now. Just a thought:
Heavy db-calls in API routes might require taylored rate-limiting to prevent DoS attacks?

Reply:
fixed by @pcrespov